src/components/Dialogs/Dialogs.jsx

```diff
@@ -1,52 +1,46 @@
 import React from 'react';
-import styles from './Dialogs.module.css';
-import DialogItem from "./DialogItem/DialogItem";
-import Message from "./Message/Message";
-import { maxLengthCreator } from '../../utils/validation/validation';
-import { AddMessageForm } from './Message/AddMessageForm';
 import { reduxForm } from 'redux-form';
+
+import DialogItem from './DialogItem/DialogItem';
+import Message from './Message/Message';
+import { AddMessageForm } from './Message/AddMessageForm';
 import Sidebar from '../Sidebar/Sidebar';
 import Navbar from '../Navbar/Navbar'
-
-const maxLength10 = maxLengthCreator(10);
+import styles from './Dialogs.module.css';

```

Comment:
same applies here, since in your Dialogs/ directory is only 1 css module file, you can easily call it:
```suggestion
import styles from './styles.module.css';

```